### ISSUE TITLE
[Suggestion] Battle tower daily content or trainers/ npc gyms overworld

### ISSUE THREAD

**Issue by xWattzz** [NebulaMC-GG/Suggestions#97]
implementation of a battle tower that's completable daily for rewards like bingo? 
or just trainers and npc gyms scattered in overworld would keep people active and give something to do

honestly bingo is a chore and once your dex is completed you don't wanna go out and catch a load of stuff to just fill boxes again with nonsense for measly 20k I haven't seen any calls for bingo in chat since changes no one really does it anymore.

The battle tower could have like 5 to 10 floors each floor gain a reward getting better in rarity or whatever and gives people a reason to use there mons they've grinded to get or bred to use, because there is honestly not any battles unless barely ever a good boss might spawn on you and is still most likely getting 1 shot.

I understand adventure mode is coming whenever but once completed will we not just end up back in overworld with nothing to battle again? I also understand new players still have alot to do with bingo and dex those things there but i also know like most of my server just stand around either waiting on a legend spawn not even really playing just afk or the others are doing the same just at their breeders, would give end game players something to do? just a thought just would be nice to have some real battles.

Spoken to a few people and they think its a good idea you can probably see master's play times and how they dropped after hitting master its hard to stay dedicated when there is nothing to use anything on right now.

Also a fix on the transfer tutor would be beautiful since he's been bodged for like a month maybe even before but I wasn't here to know.

**Comment by Khrisstias** [comment]
Battle Tower has now been added to the server! Thank you for the suggestion.
